strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/api/impl/StrongboxBuiltinAuthenticationSupplier.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.authentication.api.impl;
+
+import org.carlspring.strongbox.authentication.api.AuthenticationSupplier;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.servlet.http.HttpServletRequest;
+import java.io.UnsupportedEncodingException;
+
+import com.google.common.base.Throwables;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.codec.Base64;
+import org.springframework.util.Assert;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class StrongboxBuiltinAuthenticationSupplier
```

Comment:
Is this using the XML file, or OrientDB? The name doesn't look very good... I think it should be something along the lines of `OrientDbAuthenticationSupplier`, or `XLAuthenticationSupplier`, or `DefaultAuthenticationSupplier`...